.github/actions/next-stats-action/src/constants.js

```diff
@@ -1,12 +1,10 @@
 const path = require('path')
-const os = require('os')
+const fs = require('fs')
 
 const benchTitle = 'Page Load Tests'
-const workDir = path.join(os.tmpdir(), 'next-stats')
-const mainRepoName = 'main-repo'
-const diffRepoName = 'diff-repo'
-const mainRepoDir = path.join(workDir, mainRepoName)
-const diffRepoDir = path.join(workDir, diffRepoName)
+const workDir = path.join(fs.mkdtempSync(), 'next-stats')
```

Comment:
```suggestion
const workDir = path.join(os.tmpdir(), fs.mkdtempSync('next-stats'))
```

Reply:
Oh, I'm a little confused by the docs, shouldn't it be like this?
```
fs.mkdtempSync(path.join(os.tmpdir(), 'next-stats'))
```
Since it actually creates the file, and creating at `/` would throw locally (at least on my system and I think that OSX has also `/` read-only)

Reply:
https://nodejs.org/api/fs.html#fsmkdtempprefix-options-callback

Reply:
@JanKaifer Ah yes, you are correct